src/views/Run/internal/ResultDisplay/internal/ChartsSection/internal/BarChart/BarChart.tsx

```diff
@@ -0,0 +1,42 @@
+import { ResponsiveBarCanvas } from '@nivo/bar'
+import { FC } from 'react'
+
+import { getClassNames } from './internal/BarChart.helpers'
+import { IProps } from './internal/BarChart.types'
+
+export const BarChart: FC<IProps> = ({ className, data, xLegend, yLegend }) => {
+  const classNames = getClassNames({ className })
+
+  const commonProps = {
+    // Comment if you want to hide the axis legends
+    // Necessary if you want them to be shown
+    margin: { left: 60, bottom: 60 },
+  }
+
+  const axisLeft = {
+    legend: yLegend,
+    legendOffset: -40,
+  }
+
+  const axisBottom = {
+    legend: xLegend,
+    legendOffset: 40,
+  }
+
+  return (
+    <div className={classNames.join(' ')} style={{ height: '350px' }}>
+      <ResponsiveBarCanvas
+        {...commonProps}
+        colors={{ scheme: 'dark2' }}
```

Comment:
Very nice! I would only suggest to use the primary color in the Barchart component :

```suggestion
        colors="#f1b445"
```

Reply:
Yes, indeed!
I thought it would be more difficult to change, with a scheme, but it seems easy enough on this kind of chart with only one group.
However, I am not sure what color to choose for the labels in this case.
I feel like grey is more readable than white, but it is hardy visible when the column is very small (like in the right). Which one seems better, or do you have another idea?
![Screenshot from 2022-10-12 12-32-06](https://user-images.githubusercontent.com/74239400/195320415-0f27b946-03c6-4b51-a1ba-f788408c6f8e.png)
![Screenshot from 2022-10-12 12-32-33](https://user-images.githubusercontent.com/74239400/195320428-f988bf3c-87b9-4a84-b78e-69b103ed7894.png)